test/config/config.go

```diff
@@ -193,6 +196,15 @@ func GetDomainSuffix() string {
 	return env.domainSuffix
 }
 
+// GetImageCacheMode returns image cache mode
+func GetImageCacheMode() ImageCacheMode {
+	imageCacheMode := ImageCacheMode(env.imageCacheMode)
+	if imageCacheMode.isValid() {
+		return imageCacheMode
+	}
+	panic(fmt.Errorf("Invalid image cache mode: %s", imageCacheMode))
```

Comment:
I think this will fail when the test is running. In order to make it fail the sooner the better, we could check this in the [check_setup.go](https://github.com/kiegroup/kogito-cloud-operator/blob/master/test/framework/check_setup.go)